build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "uk.nhs.tis.trainee"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
Bump this up to `1.0.0` while you're here, I don't think we can consider it pre-release anymore.